docs/develop.rst

```diff
@@ -79,15 +79,15 @@ You can also print some useful advice for the developer here, as we do in the ``
 Template Registration
 =====================
 
-Even though you can use bobtemplates without regitration, we shoudl register the template, to allow plonecli and future mrbob version to query for it. The registration is done by adding en Python entry point into the setup.py of bobtemplates.plone or your own custom bobtemplates package and by adding a short method to the bobregistry.py file.
-
+Even though you can use bobtemplates without registration, we should register the template to allow plonecli and future mrbob versions to query for it.
+The registration is done either by adding a Python entry point into the ``setup.py`` of ``bobtemplates.plone`` or by adding your own custom ``bobtemplates`` package with a short method to the ``bobregistry.py`` file.
 Let's look first on the entry point:
 
 .. code-block:: python
 
     entry_points={
         'mrbob_templates': [
-            'plone_addon = bobtemplates.plone.bobregistry:plone_addon',
+            'plone_add-on = bobtemplates.plone.bobregistry:plone_add-on',
```

Comment:
My mistake, I did not expect you to change instances of "addon" with "add-on" within code, but only within narrative text. Use care when doing find/replace. Revert the above.